transaction/src/view.rs

```diff
@@ -0,0 +1,16 @@
+pub mod action_view;
+mod transaction_perspective;
+
+pub use action_view::ActionView;
+use decaf377_fmd::Clue;
+use penumbra_crypto::{memo::MemoPlaintext, transaction::Fee};
+pub use transaction_perspective::TransactionPerspective;
+
+pub struct TransactionView {
+    pub actions: Vec<ActionView>,
+    pub expiry_height: u64,
+    pub chain_id: String,
+    pub fee: Fee,
+    pub fmd_clues: Vec<Clue>,
+    pub memo: Option<MemoPlaintext>,
```

Comment:
This could just be `Option<String>` instead; it's less code to parse the protos, and I think we should get rid of `MemoPlaintext` eventually anyways.